src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/RequestHeaderValueResolver.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Controller\ArgumentResolver;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Attribute\MapRequestHeader;
+use Symfony\Component\HttpKernel\Controller\ValueResolverInterface;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+final class RequestHeaderValueResolver implements ValueResolverInterface
+{
+    public function resolve(Request $request, ArgumentMetadata $argument): iterable
```

Comment:
Because `RequestHeaderValueResolver` implements [ValueResolverInterface](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/HttpKernel/Controller/ValueResolverInterface.php) with the `resolve` method who has a return type iterable.

Reply:
Ha yes my bad!